lib/graphql/anycable/config.rb

```diff
@@ -12,6 +12,9 @@ class Config < Anyway::Config
       attr_config use_redis_object_on_cleanup: true
       attr_config use_client_provided_uniq_id: true
       attr_config redis_prefix: "graphql" # Here, we set clear redis_prefix without any hyphen. The hyphen is added at the end of this value on our side.
+
+      attr_config use_async_broadcasting: true
+      attr_config async_broadcasting: { queue: "broadcasting", class: "GraphQL::Adapters::BaseJob" }
```

Comment:
This is not how adapter pattern is implemented. The API must be as follows:

```ruby
GraphQL::AnyCable.delivery_method = :inline

GraphQL::AnyCable.delivery_method = :active_job

# or with options

GraphQL::AnyCable.delivery_method = :active_job, queue: "custom_queue", job_class: "MyJob"
```

And then, in the `#trigger` method we simply delegate the execution to the configured adapter instance. Smth like:

```ruby
def trigger(...)
  AnyCable.delivery_adapter.trigger(...)
end
```

Reply:
I did it, but without "="

It works like that 
```
GraphQL::AnyCable.delivery_method("active_job", queue: "custom_queue", job_class: "MyJob")
```

But, if we need like `setter`, we will able to call it only like that (adds `{` and `}` for options)

```
GraphQL::AnyCable.delivery_method = :active_job, {queue: "custom_queue", job_class: "MyJob"}
```

Because the setter, as I know, cannot accept keyword arguments

If this is okay, I will change it (or I can remain the current way if it is okay)

Reply:
> if we need like setter, we will able to call it only like that


Yes, and that's how it should be done. That's a common pattern used by Rails and many libraries, We shouldn't reinvent it.

So, please, implement it using the original proposed interface.

P.S. See, for example: https://github.com/palkan/active_delivery/blob/master/lib/abstract_notifier/async_adapters.rb